lib/ProductOpener/APIProductWrite.pm

```diff
@@ -217,6 +281,66 @@ sub update_product_fields ($request_ref, $product_ref) {
 
 			update_field_with_0_or_1_value($request_ref, $product_ref, $field, $value);
 		}
+		# language fields
+		elsif ( ($field =~ /^(.*)_(\w\w)$/)
```

Comment:
So we stick to using _xx in API v3 ?

This kind of pattern is not really OpenAPI friendly as it force to have a lot of patternProperties

Reply:
Discussed on weekly meeting. API v3 is experimental, in v3.1 we will break this to make it better.